tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/TestingFrameworkEvpTest.cs

```diff
@@ -19,32 +19,47 @@ namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI;
 
 public abstract class TestingFrameworkEvpTest : TestHelper
 {
+    private readonly GacFixture _gacFixture;
+
     protected TestingFrameworkEvpTest(string sampleAppName, string samplePathOverrides, ITestOutputHelper output)
         : base(sampleAppName, samplePathOverrides, output)
     {
         SetCIEnvironmentValues();
+        _gacFixture = new GacFixture();
+        _gacFixture.AddAssembliesToGac();
```

Comment:
Interesting, which scenarios need this? .NET FX? 🤔 Does this have implications for when customers are using `dd-trace`? 🤔

Reply:
yup, .NET FX. `dd-trace` handles that automatically, if Windows is detected it tries to do the GAC installation (https://github.com/DataDog/dd-trace-dotnet/pull/5129).